lib/dslash_ndeg_twisted_clover_preconditioned.cu

```diff
@@ -0,0 +1,156 @@
+#ifndef USE_LEGACY_DSLASH
```

Comment:
this can be removed as the legacy dslash has been evicted from the code completely